CONTRIBUTING.md

```diff
@@ -189,6 +189,46 @@ Take a look at the sections that follow to pick a route (or five) that appeal to
 
 If you ever find yourself at a loss for what to do, or in need of mentorship or advice on how to contribute to Bevy, feel free to ask in [Discord] and one of our more experienced community members will be happy to help.
 
+### Join a working group
+
+Active initiatives in Bevy are organized into temporary working groups: choosing one of those and asking how to help can be a fantastic way to get up to speed and be immediately useful.
+
+Working groups are public, open-membership groups that work together to tackle a broad-but-scoped initiative.
+The work that they do is coordinated in a forum-channel on [Discord](https://discord.gg/bevy), although they also create issues and may use project boards for tangible work that needs to be done.
+
+There are no special requirements to be a member, and no formal membership list.
+Working groups are *spaces*, not clubs.
+
+### Start a working group
+
+When tackling a complex initiative, friends and allies can make things go much more smoothly.
+
+To start a working group:
+
+1. Decide what the working group is going to focus on. This should be tightly focused and achievable!
+2. Gather at least 3 people including yourself who are willing to be in the working group.
+3. Ping the `@Maintainer` role on Discord in #engine-dev announcing your mutual intent and a one or two sentence description of your plans.
+
+The maintainers will briefly evaluate the proposal in consultation with the relevant SMEs and give you a thumbs up or down on whether this is something Bevy can and wants to explore right now.
+You don't need a concrete plan at this stage, just a sensible argument for both "why is this something that could be useful to Bevy" and "why there aren't any serious barriers in implementing this in the near future".
+If they're in favor, a maintainer will create a forum channel for you and you're off to the races.
+
+Your initial task is writing up a design doc: laying out the scope of work and general implementation strategy.
```

Comment:
I think it would be valuable to those interested if they could see an example of a design doc. 

Do we have any available that we could link to? Maybe we could just create a dummy one? Or, better yet, a template for them to fill out?

<details>
<summary>Relevant Office Meme</summary>

<img alt="The Office - What's a Rundown?" src="https://github.com/bevyengine/bevy/assets/49806985/4dfde630-30c6-460c-aa2d-803d1a9cac0c">


</summary>

Reply:
The bevy_mod_picking one is solid! Linking. I'd like to avoid templates and process here, to avoid recreating challenges with RFCs.